week-06-intro-ruby-on-rails/day-04/module-02/readme.md

```diff
@@ -1 +1,218 @@
+reichel.save
+```
+or
+```ruby
+reichel = Speaker.create({first: "Reichel", last: "Theresia", email: "giovani@gmail.com"})
+```
+
+We can also query for `all` the records in our pretend "database":
+```ruby
+puts Speaker.all
+```
+
+**Stretch**: Can you model both a `Speaker` and a `Talk` object, using our simplified HacktiveRecord? (Hint: you may need to specify `speaker_id` on your talk objects so you know who they belong to!)
+
+#Ready for Rails!
+This has been a very light introduction to faking data, seeding your database, and working with models and inheritance. We're ready to dive into rails and work with ActiveRecord!
+
+####1. Create a new rails application
+**Make sure you're in a new project folder**
+``` bash
+rails new conference_app -T -d postgresql
+cd conference_app
+bundle
+git init
+git add . -A
+git commit -m "inital commit, rails skeleton"
+subl .
+```
+
+####2. Create a `Talk` model
+``` bash
+rails g model talk topic:string duration:integer
+git diff # take a look at the files you just created!
+```
+
+And take a look at the following files:  
+* `app/models/talk.rb`
+* `db/migrate/1234566789_create_talks.rb`
+
+####3. Setup your database
+Download and Launch Postgres.app. You should see an elephant in your menu if it's running.
+
+Next, create your application database:
+```bash
+rake db:create # create a new database on your machine
+rake db:migrate # instruct your database what tables it needs to contain
+```
+
+####4. Launch the rails console and create your first talk!
+```bash
+rails console
+# or
+rails c
+```
+
+**Confirm that your model exists**  
+```ruby
+Talk
+ #=> Talk(id: integer, topic: string, duration: integer, created_at: datetime, updated_at: datetime)
+```
+
+(You may need to "connect" to your database in the rails console. Just follow the instructions.)
+
+**Create your first record**!
+``` ruby
+Talk.create({topic: "Playing with Models in the Rails Console"})
+Talk.count
+Talk.all
+```
+
+**Try the following:**  
+* Create 3 new Talks in the Rails Console.
+* Delete the last talk you created
+* Find the first talk
+* Find the last talk
+* Search by id
+* Search by title
+* Sort by title
+* Update the title of the last talk you created
+* Delete all the talks you created.
+
+**Pro-Tip**: Remember, when you're working in the console/repl `up-arrow` is your friend! (That and "hanging a dot" + "double-tabbing").
+
+####5. Can you seed your database?
+Take a look at `db/seed.rb`.
+
+Add the following line:
+```ruby
+p "Hello from seed.rb"
+```
+
+Now run the following from your command line (not the console!):
+```bash
+rake db:seed
+# Hello from seed.rb
+```
+
+The `seed.rb` file is magic, because it _already_ knows about all of the models and gems in your rails app. All you have to do is tell it what data to create!
+
+**Can you seed your database with speaker and talks?**
+
+**Bonus**: can you do it with the FFaker gem?
+
+### Homework:
```

Comment:
@ben-manning Instead of putting HW here, can you make a separate repo for it with a short description and the link to Rails for Zombies. Even though it's an outside tutorial, we still need to have "Getting Started" and "Submission" instructions. They should still turn something in every night -- maybe have them create a PR with questions that came up during the tutorial?